resolwe_bio/processes/variant_calling/filtering_chemut.yml

```diff
@@ -11,12 +11,13 @@
       docker:
         image: resolwebio/dnaseq:4.1.0
   data_name: "Filtered variants ({{ analysis_type }})"
-  version: 1.3.1
+  version: 1.4.0
   type: data:variants:vcf:filtering
   category: Other
   persistence: CACHED
   description: |
-    Filtering and annotation of Variant Calling data - Chemical mutagenesis in Dictyostelium discoideum
+    Filtering and annotation of Variant Calling data - Chemical
+    mutagenesis in Dictyostelium discoideum
```

Comment:
Can we use markdown in here, i.e. _Dictyostelium discoideum_?

Reply:
We can, so I will change this to `_Dictyostelium discoideum_`